tests/test_isrTaskLSST.py

```diff
@@ -766,17 +866,17 @@ def get_isr_config_electronic_corrections(self):
         isr_config.doBias = False
         isr_config.doDark = False
         isr_config.doFlat = False
+        isr_config.doBrighterFatter = False
```

Comment:
Please add a comment here about why we have this `False` here.  Also I think that this should be moved to line 875 (electronic effects that the tests support), but it is `False` because it is only used in a specialized tests.